src/index.ts

```diff
@@ -7,12 +7,15 @@ import { setFindexInit } from "./findex/findex"
 import fpe_wasm from "./pkg/fpe/cloudproof_fpe_bg.wasm"
 import { setFpeInit } from "./fpe/fpe"
 
+import anonymization_wasm from "./pkg/anonymization/cloudproof_anonymization_bg.wasm"
+import { setAnonymizationInit } from "./anonymization/anonymization"
+
 export {
   CoverCryptMasterKey,
   type CoverCryptKeyGeneration,
 } from "./cover_crypt/key_generation"
-export { type CoverCryptHybridDecryption } from "./cover_crypt/decryption"
-export { type CoverCryptHybridEncryption } from "./cover_crypt/encryption"
+export { type CoverCryptHybridDecryption } from "./cover_crypt/decryption" // to remove?
+export { type CoverCryptHybridEncryption } from "./cover_crypt/encryption" // to remove?
```

Comment:
Why "to remove"?

Reply:
since we have this:
```export * from "./cover_crypt/cover_crypt"``` and that the function `CoverCrypt` is then exported, I wondered if it was still required to export those 2 lines

Reply:
`"./cover_crypt/cover_crypt"` doesn't export these types. Maybe it works if `"./cover_crypt/cover_crypt"` exports the types but not sure about type export forwarding…